build-scripts/components/kubernetes/pre-patches/0000-Kubelite-integration.patch

```diff
@@ -397,10 +398,10 @@ index 00000000000..667b24f68e6
 +	println("Stopping kubelite")
```

Comment:
this looks new, is it needed?

Reply:
This must be a github behavior. Maybe the having a patch in the diff between PRs breaks the presentation. The line in question, for example, already exists, it is not new: https://github.com/canonical/microk8s/blob/master/build-scripts/components/kubernetes/pre-patches/0000-Kubelite-integration.patch#L397